source/numerics.tex

```diff
@@ -1786,6 +1786,30 @@
 \indextext{requirements!uniform random bit generator|)}%
 \indextext{uniform random bit generator!requirements|)}
 
+\pnum
+The \tcode{UniformRandomBitGenerator} concept is a slight relaxation of the uniform random bit generator requirements, in that it does not require a nested \grammarterm{typedef-name} \tcode{result_type}.
+
```

Comment:
The presentation seems suboptimal and redundant here. It seems more compact to introduce \concept{UniformRandomBitGenerator} first (which is the more relaxed concept) and then define "uniform random bit generator" in terms of that, adding the result_type requirement.

Reply:
You mean something like [urbg.pdf](https://github.com/cplusplus/draft/files/2117498/urbg.pdf)? That seems to go a bit past "editorial", but I'm game if you folks are.

Reply:
It also relaxes the rules by only requiring the result type to model `UnsignedIntegral` instead of be an unsigned integer type. The former permits `bool` and character types that are unsigned.

Reply:
I'll reword to:
```tex
A class \tcode{G} meets the \term{uniform random bit generator} requirements if
\tcode{G} models \libconcept{UniformRandomBitGenerator},
\tcode{invoke_result_t<G\&>} is an unsigned integer type\iref{basic.fundamental},
and
\tcode{G} provides a nested \grammarterm{typedef-name} \tcode{result_type}
that denotes the same type as \tcode{invoke_result_t<G\&>}.
```
to fix the normative difference between "unsigned integer type" and `UnsignedIntegral` and add this in a separate commit.